scripts/predefined_completion.py

```diff
@@ -0,0 +1,89 @@
+"""
+This script is used to generate an Ada file containing an hard-coded
+JSON string that contains all the information for predefined completion
+items (pragmas, attributes and aspects).
+
+The script should be laucnhed like this:
+
+   python predefined_completion.py <json_filename>
+
+Where <json_filename> points to a JSON file with the following format:
+
+{
+"PREDEFINED_ADA": {
+  "ASPECT": [
+    {
+      "DOC": "This aspect is equivalent to *note pragma Abstract_State: 1c.",
+      "_id": "0",
+      "_name": "Abstract_State",
+      "_origin": "GNAT RM",
+      "_category": "unknown"
+   },
+   ...
+  "ATTRIBUTE": [
+    {
+      "DOC": "Doc for Abort_Signal pragma",
+      "_id": "0",
+      "_name": "Abort_Signal",
+      "_origin": "GNAT RM",
+      "_category": "variable"
+   },
+   ...
+   "PRAGMA": [
+     {
+       "DOC": "Doc for Abort_Defer pragma",
+       "_id": "0",
+       "_name": "Abort_Defer",
+       "_origin": "GNAT RM"
+   },
+   ...
+}
+
+JSON files of this format can be directly retrieved by converting GNAT Studio's
```

Comment:
This comment is related to GS, not sure it applies here

Reply:
Ok, I'll remove it then.